src/components/GuestActions/MethodList.tsx

```
@@ -52,13 +54,30 @@ const GUEST_ACTION_METHODS: Method[] = [
+    } = useGuestFlow()
 
-    const handleMethodClick = (method: Method) => {
+    const handleMethodClick = async (method: Method) => {
         switch (method.id) {
             case 'bank':
-                setGuestFlowStep('bank-country-list')
+                if (!claimLinkData.sender?.userId) {
+                    setShowVerificationModal(true)
+                    return
+                }
+                const senderDetails = await getUserById(claimLinkData.sender?.userId ?? claimLinkData.senderAddress)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)